sources/packages/backend/libs/services/src/workflow/workflow-client.service.ts

```diff
@@ -32,7 +32,7 @@ export class WorkflowClientService {
         `Error while starting application assessment workflow: ${workflowName}`,
       );
       this.logger.error(error);
-      // The error is not thrown here, as we are failing silently.
+      throw error;
```

Comment:
Is it the final decision to throw the error here or are we still pending a conversation with @abschwenker about it?